sdk/identity/azure-identity/tests/test_obo_async.py

```diff
@@ -70,8 +70,6 @@ async def test_context_manager():
 
 @pytest.mark.asyncio
 async def test_allow_multitenant_authentication():
```

Comment:
Suggest renaming these now that there's no such argument to test:
```suggestion
async def test_multitenant_authentication():
```